config/deploy/prod.rb

```diff
@@ -1,10 +1,6 @@
 # frozen_string_literal: true
 
-server 'sul-robots1-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
-server 'sul-robots2-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots3-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots4-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
-server 'sul-robots5-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app}
+server 'common-accessioning-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db} # only server to get whenever cronjobs deployed to
```

Comment:
Hmm... I thought Tony was going to make 2 servers for this. 😞

Reply:
We started a `-prod` and a `-stage` initially, we could add more I assume.

Reply:
I think it's important we have stage-a/b so that we demonstrate we can divide the load.